<launch>
    <node name="map_traverser" pkg="wifi_localization" type="map_traverser" output="screen">
        <param name="p1_x" type="double" value="-5.8" />
        <param name="p1_y" type="double" value="7.3" />
        <param name="p2_x" type="double" value="-4.9" />
        <param name="p2_y" type="double" value="7.45" />
        <param name="p3_x" type="double" value="-4.5" />
        <param name="p3_y" type="double" value="5.6" />
        <param name="ordered" type="bool" value="true" />
    </node>
</launch>